pom.xml

```diff
@@ -74,6 +74,7 @@
         <groupId>org.jenkins-ci.tools</groupId>
         <artifactId>maven-hpi-plugin</artifactId>
         <configuration>
+          <compatibleSinceVersion>1.15</compatibleSinceVersion>
```

Comment:
Yes, I would prefer at a minimum to have a comment explaining why beside this entry in the Pom so that future explorers can understand the reasoning